app/components/item/description.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
A convention of the project is to capitalize filenames of files which have react components. Something small.